imports/plugins/core/ui/client/components/button/buttonSelect.js

```diff
@@ -128,6 +128,7 @@ class ButtonSelect extends Component {
           {nonActiveButtons.map((button, key) => {
             return (<div
               className="button-item" key={key}
+              data-i18n={button.i18nKeyLabel}
```

Comment:
- dont use `data-i18n` attributes, they are of a jQuery/blaze legacy which is going away
- fix the indentation
- use a button component